attach/recvattach.c

```diff
@@ -553,28 +553,25 @@ static void query_pipe_attachment(const char *command, FILE *fp, struct Body *bo
       msgwin_clear_text();
       return;
     }
-    mutt_mktemp(tfile, sizeof(tfile));
-  }
-  else
-  {
-    tfile[0] = '\0';
```

Comment:
I didn't understand why need to NULL terminate here. Please double check.

Reply:
You're right, this isn't needed any more.
It was made redundant by commit 7380cb5fae63481d5d8c10c9a669ebfafcd80ef3 (a year ago), which initialises `tfile`.